tests/model_forms/tests.py

```diff
@@ -847,7 +847,9 @@ class Meta:
         self.assertEqual(m1.mode, mode)
 
     def test_renderer_kwarg(self):
-        custom = object()
+        from django.forms.renderers import BaseRenderer
+
+        custom = type("custom", (BaseRenderer,), {})()
```

Comment:
ok, so I don't really understand what's going on here. 

I'm slightly concerned that the fact this test is changing means that we're introducing a backward incompatible change.

Reply:
The problem is [this change](https://github.com/django/django/pull/18266/files#diff-bec9cad724e24202d0a304c98660b17539c743415ad41b05bdbf6508781798b6R130-R134). Prior to that, the rendered was unused in `__init__` so it didn't have to be an actual renderer.

Reply:
@smithdc1 Are you happy with the proposed explanation?